scaling_performance/using_cluster_loader.adoc

```diff
@@ -0,0 +1,245 @@
+[[scaling-performance-using-cluster-loader]]
+= Using Cluster Loader
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[what-cluster-loader-does]]
+== What Cluster Loader Does
+
+Cluster Loader is a tool that deploys large numbers of various objects to a
+cluster, which creates user-defined cluster objects. Build, configure, and run
+Cluster Loader to measure performance metrics of your {product-title} deployment
+at various cluster states.
+
+[[installing-cluster-loader]]
+== Installing Cluster Loader
+
+Cluster Loader is included in the `atomic-openshift-tests` package. To install
+it, run:
+
+----
+$ yum install atomic-openshift-tests
+----
+
+After installation, the test executable `extended.test` is located in
+*_/usr/libexec/atomic-openshift/extended.test_*.
+
+[[running-cluster-loader]]
+== Running Cluster Loader
+
+. Set the `KUBECONFIG` variable to the location of the administrator `kubeconfig`:
++
+----
+$ export KUBECONFIG=${KUBECONFIG-$HOME/.kube/config}
+----
+
+. Execute Cluster Loader using the built-in test configuration, which deploys five
+template builds and waits for them to complete.
++
+----
+$ cd /usr/libexec/atomic-openshift/
+./extended.test --ginkgo.focus="Load cluster"
+----
++
+Alternatively, execute Cluster Loader with a user-defined configuration by
+adding the flag for `--viper-config`:
++
+----
+$ ./extended.test --ginkgo.focus="Load cluster" --viper-config=config/test <1>
+----
+<1> In this example, there is a subdirectory called `config` with a configuration
```

Comment:
`config` would be marked up like a file name too, I think.

Reply:
Also add a trailing `/` like in later directory references.

Reply:
Good catch!